File: src/main/Java/boj_step3/boj10951.java
import java.io.*;
import java.util.StringTokenizer;

public class boj10951 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        try {
            while (true) {
                String string = reader.readLine();
                StringTokenizer tokenizer = new StringTokenizer(string);
                int a = Integer.parseInt(tokenizer.nextToken());
                int b = Integer.parseInt(tokenizer.nextToken());
                writer.append(String.valueOf(a + b));
                writer.newLine();
                writer.flush();
            }
        } catch (Exception e) { e.getMessage(); }
    }
}

File: src/main/kotlin/boj10951.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.lang.Exception

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    try {
        while (true) {
            val s = r.readLine()
            if (s.isNullOrEmpty()) break
            else {
                val (a, b) = s.split(" ").map { it.toInt() }
                w.write((a + b).toString() + "\n")
            }
        }
        w.close()
    } catch (e: Exception) {
        e.printStackTrace()
    }
}